### PR TITLE
Ft seller update product

### PR DESCRIPTION
**Pull request by rodriguecyber**
### What does this PR 
seller update his/her product

### How should this be manually tested?

- click on single product
- fill popup box
- 

#### N/A

### CODE REVIEW

**Review by Ishimwe7 — APPROVED** [review]
Good job @rodriguecyber , this PR is ready to be merged

**Review by derrick-nuby — COMMENTED** [review]
This feature was implemented as expected; I tested it and it ready for merge

**Review by derrick-nuby — APPROVED** [review]
Ready for Merge

**Review by kirengamartial — APPROVED** [review]
The code is working really well and it is ready for merge

**Review by Balinda21 — APPROVED** [review]
Well implemented feature, PR is ready to be merged!

File: src/utils/api.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import Cookies from "js-cookie";
+
+export const api = axios.create({
+  baseURL: `https://team-lydia-demo.onrender.com`,
+  headers: {
+    "Content-Type": "application/json",
+  },
+});
+
```
Comment (robsdagreat) [inline src/utils/api.ts:10]: well done @rodriguecyber , these changes were needed.

**Review by robsdagreat — APPROVED** [review]

**Review by bahati10 — APPROVED** [review]
Ready to be merged, Good job on this one.

**Review by DeliceLydia — APPROVED** [review]

**Pull request merged**